apis/kubedb/v1alpha2/perconaxtradb_types.go

```diff
@@ -126,13 +126,22 @@ type PerconaXtraDBSpec struct {
 	SystemUserSecrets *SystemUserSecretsSpec `json:"systemUserSecrets,omitempty"`
 }
 
-// +kubebuilder:validation:Enum=server;client;metrics-exporter
+// +kubebuilder:validation:Enum=server;client;exporter
 type PerconaXtraDBCertificateAlias string
 
 const (
-	PerconaXtraDBServerCert          PerconaXtraDBCertificateAlias = "server"
-	PerconaXtraDBClientCert          PerconaXtraDBCertificateAlias = "client"
-	PerconaXtraDBMetricsExporterCert PerconaXtraDBCertificateAlias = "metrics-exporter"
+	PerconaXtraDBServerCert   PerconaXtraDBCertificateAlias = "server"
+	PerconaXtraDBClientCert   PerconaXtraDBCertificateAlias = "client"
+	PerconaXtraDBExporterCert PerconaXtraDBCertificateAlias = "exporter"
+)
+
+// +kubebuilder:validation:Enum=ca.crt;tls.crt;tls.key
+type PerconaXtraDBCertFileNameAlias string
+
+const (
+	PerconaXtraDBCACertFileName  PerconaXtraDBCertFileNameAlias = "ca.crt"
+	PerconaXtraDBTLSCertFileName PerconaXtraDBCertFileNameAlias = core.TLSCertKey
+	PerconaXtraDBTLSKeyFileName  PerconaXtraDBCertFileNameAlias = core.TLSPrivateKeyKey
```

Comment:
Do we use this in an api type? If not, we don't need the enum. core.TLSCertKey and core.TLSPrivateKeyKey are already well-known constants.

Reply:
We don't use this on API type. Removed enum.